bin/mailing-2-8-teacher-recruitment/state_abbr_to_name.rb

```diff
@@ -0,0 +1,53 @@
+# Adapted from http://snipplr.com/view/35407/ruby-state-abbreviations/
+STATE_ABBR = {
+  'AL' => 'Alabama',
+  'AK' => 'Alaska',
+  'AZ' => 'Arizona',
+  'AR' => 'Arkansas',
+  'CA' => 'California',
+  'CO' => 'Colorado',
+  'CT' => 'Connecticut',
+  'DE' => 'Delaware',
+  'FL' => 'Florida',
+  'GA' => 'Georgia',
+  'HI' => 'Hawaii',
+  'ID' => 'Idaho',
+  'IL' => 'Illinois',
+  'IN' => 'Indiana',
+  'IA' => 'Iowa',
+  'KS' => 'Kansas',
+  'KY' => 'Kentucky',
+  'LA' => 'Louisiana',
+  'ME' => 'Maine',
+  'MD' => 'Maryland',
+  'MA' => 'Massachusetts',
+  'MI' => 'Michigan',
+  'MN' => 'Minnesota',
+  'MS' => 'Mississippi',
+  'MO' => 'Missouri',
+  'MT' => 'Montana',
+  'NE' => 'Nebraska',
+  'NV' => 'Nevada',
+  'NH' => 'New Hampshire',
+  'NJ' => 'New Jersey',
+  'NM' => 'New Mexico',
+  'NY' => 'New York',
+  'NC' => 'North Carolina',
+  'ND' => 'North Dakota',
+  'OH' => 'Ohio',
+  'OK' => 'Oklahoma',
+  'OR' => 'Oregon',
+  'PA' => 'Pennsylvania',
+  'RI' => 'Rhode Island',
+  'SC' => 'South Carolina',
+  'SD' => 'South Dakota',
+  'TN' => 'Tennessee',
+  'TX' => 'Texas',
+  'UT' => 'Utah',
+  'VT' => 'Vermont',
+  'VA' => 'Virginia',
+  'WA' => 'Washington',
+  'WV' => 'West Virginia',
+  'WI' => 'Wisconsin',
+  'WY' => 'Wyoming'
+}
```

Comment:
Good suggestion. I moved it into mailing-common.

I also prefer the newer hash syntax, but it only works with symbols as keys not strings. {'WY': 'Wyoming'} will convert the 'WY' to :WY. However since symbols are preferred for hash keys anyway, I went with that and added a lookup method to call .to_sym on the provided string before checking the hash.

Updated.